context/ue.go

```diff
@@ -68,6 +65,9 @@ type UeAMPolicyData struct {
 	AmPolicyData *models.AmPolicyData // Svbscription Data
 	// Corresponding UE
 	PcfUe *UeContext
+	// Policy Association
+	Triggers []models.RequestTrigger
+	Rfsp     int32
```

Comment:
It seems to me that the `// Policy Association` comment is about multiple variables. What about doing something like this to keep those variable still together?

```diff
diff --git a/context/ue.go b/context/ue.go
index 3acfa26..33b2218 100644
--- a/context/ue.go
+++ b/context/ue.go
@@ -55,19 +56,19 @@ type UeAMPolicyData struct {
        Guami        *models.Guami
        ServiveName  string
        // TraceReq *TraceData
-       // Policy Association
-       Triggers    []models.RequestTrigger
-       ServAreaRes *models.ServiceAreaRestriction
-       Rfsp        int32
-       UserLoc     *models.UserLocation
-       TimeZone    string
-       SuppFeat    string
        // about AF request
        Pras map[string]models.PresenceInfo
        // related to UDR Subscription Data
        AmPolicyData *models.AmPolicyData // Svbscription Data
        // Corresponding UE
        PcfUe *UeContext
+       // Policy Association
+       ServAreaRes *models.ServiceAreaRestriction
+       UserLoc     *models.UserLocation
+       TimeZone    string
+       SuppFeat    string
+       Triggers    []models.RequestTrigger
+       Rfsp        int32
 }

 type UeSmPolicyData struct {
```

Reply:
done